arch/Kconfig

```diff
@@ -722,6 +722,10 @@ config ARCH_VMA_MAPPING
 	bool "Support runtime memory mapping into SHM area"
 	default n
 
+config ARCH_KVMA_MAPPING
```

Comment:
why need add additional option if we just support one type of kernel memory mapping?

Reply:
Similar to commit 70de321 (sorry I can't find the patch id for that commit),  the dependency should be from `MM` to `ARCH`, not vice versa. If this is correct understanding, then we need a thing like `ARCH_VMA_MAPPING` to select from `MM_KMAP`, and that thing further enables both `ARCH_KMAP_VBASE` and `ARCH_KMAP_NPAGES`.